modules/installation-user-defined-tags-azure.adoc

```diff
@@ -28,44 +28,47 @@ compute: <4>
+    baseDomainResourceGroupName: os4-common <10>
+    cloudName: AzurePublicCloud <11>
     outboundType: Loadbalancer
-    region: southindia <2>
-    userTags: <10>
+    region: southindia <12>
+    userTags: <13>
       createdBy: user
       environment: dev
 ----
 <1> Defines the trust bundle policy.
-<2> Required. The installation program prompts you for this value.
+<2> Required. The `baseDomain` parameter specifies the base domain of your cloud provider. The installation program prompts you for this value.
 <3> You must set the `featureSet` field as `TechPreviewNoUpgrade`.
-<4> If you do not provide these parameters and values, the installation program provides the default value.
+<4> The configuration for the machines that comprise compute. The `compute` section is a sequence of mappings. To meet the requirements of the different data structures, the first line of the `compute` section must begin with a hyphen, `-`. If you do not provide these parameters and values, the installation program provides the default value.
 <5> To enable or disable simultaneous multithreading, or `hyperthreading`. By default, simultaneous multithreading is enabled to increase the performance of your machines' cores. You can disable it by setting the parameter value to `Disabled`. If you disable simultaneous multithreading in some cluster machines, you must disable it in all cluster machines.
-<6> The `controlPlane` section is a single mapping, but the `compute` section is a sequence of mappings. To meet the requirements of the different data structures, the first line of the `compute` section must begin with a hyphen, `-`, and the first line of the `controlPlane` section must not. Only one control plane pool is used.
-<7> The cluster network plugin to install. The supported values are `OVNKubernetes` and `OpenShiftSDN`. The default value is `OVNKubernetes`.
-<8> Specifies the resource group for the base domain of the Azure DNS zone.
-<9> Specifies the name of the Azure cloud environment. You can use the `cloudName` field to configure the Azure SDK with the Azure API endpoints. If you do not provide value, the default value is Azure Public Cloud.
-<10> Defines the additional keys and values that the installer adds as tags to all Azure resources that it creates.
+<6> The configuration for the machines that comprise the control plane. The `controlPlane` section is a single mapping. The first line of the `controlPlane` section must not begin with a hyphen, `-`. You can use only one control plane pool. If you do not provide these parameters and values, the installation program provides the default value.
+<7> To enable or disable simultaneous multithreading, or `hyperthreading`. By default, simultaneous multithreading is enabled to increase the performance of your machines' cores. You can disable it by setting the parameter value to `Disabled`. If you disable simultaneous multithreading in some cluster machines, you must disable it in all cluster machines.
```

Comment:
`If you disable simultaneous multithreading in some cluster machines, you must disable it in all cluster machines.`
Does this mean you cannot disable simultaneous multithreading in just in selected cluster machines? If yes, you can mention that.